assets/src/edit-story/components/library/panes/shared.js

```diff
@@ -36,76 +31,58 @@ const Tabs = styled.ul`
   list-style: none;
 `;
 
-const Tab = styled.li`
-  width: 64px;
-  height: 100%;
-  color: ${({ theme }) => theme.colors.fg.v1};
+const Icon = styled.div`
```

Comment:
![Screenshot 2020-02-24 at 10 38 19](https://user-images.githubusercontent.com/237508/75146396-e9095a00-56f2-11ea-933f-71c14e4ac819.png)
This focus ring looks really change. It should really be 100% width and height.

Reply:
Then it'll be cut off by surrounding elements and you'll only see a small part of it. Maybe only 1, 2 or 3 sides, which makes it visually unusable.

I am very open to a better focus state in UX, but barring that, I implemented what I thought was most functional.

Reply:
@samitron7 ^ Flagged for UX review.

Reply:
@spacedmonkey @barklund I'm currently working on all the hover states now. Stay tune. Will flag when it's done.

Reply:
For now the entire tab changes color to `colors.action` to stand out as something clearly missing. Created followup in #423.

Current temporary design for selected tab (with and without focus):
![Screen Shot 2020-02-27 at 2 29 16 PM](https://user-images.githubusercontent.com/637548/75479781-19a8f800-596e-11ea-9618-d6ff66c15929.png)
![Screen Shot 2020-02-27 at 2 29 27 PM](https://user-images.githubusercontent.com/637548/75479783-19a8f800-596e-11ea-9c0c-b26c0779d733.png)